.github/workflows/runtime_eslint_plugin_e2e.yml

```diff
@@ -40,11 +40,11 @@ jobs:
         uses: actions/cache@v4
         id: node_modules
         with:
-          path: "**/node_modules"
+          path: "node_modules"
```

Comment:
Was this just to bust the cache? I don't see why what it was previously needs to be changed

Reply:
The `node_modules` within the fixtures are soft-linking to the build directory for the eslint-plugin.  When those `node_modules` were fulfilled from cache, the symlinks were broken, and after the build, it's not re-linked.  So this changes it so that it's still caching the root node_modules, but not the nested ones.  That way those node_modules with the symlinks can be created each time.